packs/feats/counterclockwork-focus.json

```diff
@@ -35,35 +35,65 @@
         },
         "rules": [
             {
-                "domain": "all",
                 "key": "RollOption",
-                "option": "counterclockwork",
+                "option": "counterclockwork-focus",
                 "toggleable": true
             },
             {
+                "disabledIf": [
+                    {
+                        "not": "counterclockwork-focus"
+                    }
+                ],
```

Comment:
This should be able to be done as one RollOption RE. A suboption will also set the base option when toggled on.

Reply:
Yeah, but treating the ammunition as a precious material is optional. I suppose I can't think of a reason you wouldn't do so right now, but that was the rationale. I can make it one toggle.

Reply:
Make a `none` option?

Reply:
Not sure why it didn't occur to me. Done!